test_utils_helpers.py

```
@@ -0,0 +1,518 @@
+#!/usr/bin/env python3
+"""
+Test suite for utils/helpers.py
+Tests all utility helper functions with comprehensive coverage
+"""
+
+import pytest
+import json
+import tempfile
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)